src/index.js

```diff
@@ -12,11 +12,9 @@ class MulticastDNS extends EventEmitter {
     options = options || {}
 
     this.broadcast = options.broadcast !== false
-    this.interval = options.interval || (1e3 * 10)
```

Comment:
@richardschneider instead of removing the feature all together, it should support for a single beacon.

Reply:
Not really sure why you want a beacon? 

`interval` controlled how often an MDNS query is sent.  Every time it is sent, all other IPFS nodes must respond with their DNS records.

Reply:
@diasdavid any comments?

Reply:
I do not see a need to drop the _interval beacon feature_. The module should support both, single beacon and interval.

Reply:
First off, a beacon is not what MDNS is about.  Secondly it can be very expensive.

Consider a data centre with 100 servers all running IPFS with an interval of 10 minutes, or 600 seconds. On average, a node will `beacon` every 6 seconds.  Then every 6 seconds all other nodes must wake-up and reply with their DNS records.  All nodes will then see all the nodes responses and have to then `.emit` discovered nodes.  It's a real waste of network bandwidth and CPU.